mode/examples/Topics/Image Processing/Blending/Blending.pyde

```diff
@@ -21,17 +18,17 @@ modes = ((REPLACE, "REPLACE"),
          (MULTIPLY, "MULTIPLY"),
          (SCREEN, "SCREEN"),
          (REPLACE, "REPLACE"))
+img1 = loadImage("layer1.jpg")
```

Comment:
Tested to make sure this works before in setup()? Some things are wonky, like loading fonts, I think, because they depend on the existence of a graphics context.